presets/4.5/bnf/droneaid.txt

```diff
@@ -93,14 +88,18 @@ vtx 2 2 0 0 4 1900 2100
      #$ OPTION END
 
      #$ OPTION BEGIN (UNCHECKED): Rush Solo without low band
-        vtxtable bands 5
+        vtxtable bands 6
         vtxtable channels 8
         vtxtable band 1 BAND_A   A CUSTOM  5865 5845 5825 5805 5785 5765 5745 5725
-        vtxtable band 2 BAND_B   B CUSTOM  5733 5752 5771 5790 5809 5828 5847 5866
-        vtxtable band 3 BAND_E   E CUSTOM  5705 5685 5665 5665 5885 5905 5905 5905
+        vtxtable band 2 BAND_B   B CUSTOM  5733 5752 5771 5790 5999 5828 5847 5866
+        vtxtable band 3 BAND_E   E CUSTOM  5705 5685 5665 5645 5885 5905 5925 5945
         vtxtable band 4 AIRWAVE  F CUSTOM  5740 5760 5780 5800 5820 5840 5860 5880
         vtxtable band 5 RACEBAND R CUSTOM  5658 5695 5732 5769 5806 5843 5880 5917
+        vtxtable band 6 LOWRACE  L CUSTOM  5362 5399 5436 5473 5510 5547 5584 5621
```

Comment:
The title of this option is "Rush Solo without low band", but you added the lowband here.